packages/zapp/console/src/components/Launch/LaunchForm/test/LaunchTaskForm.test.tsx

```diff
@@ -242,21 +242,6 @@ describe('LaunchForm: Task', () => {
       await waitFor(() => expect(submitButton).not.toBeDisabled());
     });
 
-    it('should not show validation errors until first submit', async () => {
-      const { container, getByLabelText } = renderForm();
-      const integerInput = await waitFor(() =>
-        getByLabelText(integerInputName, {
-          exact: false,
-        }),
-      );
-      fireEvent.change(integerInput, { target: { value: 'abc' } });
-
-      await waitFor(() => expect(integerInput).not.toBeInvalid());
-
-      fireEvent.click(getSubmitButton(container));
-      await waitFor(() => expect(integerInput).toBeInvalid());
-    });
-
     it('should update validation errors while typing', async () => {
```

Comment:
we need to update test description, as it's no longer checking for validation errors